src/Components/TodoPage/InputWithLabels/InputWithLabel.jsx

```diff
@@ -5,15 +5,13 @@ import PropTypes from "prop-types";
 function InputWithLabel({ children, todoTitle, handleTitleChange }) {
```

Comment:
you have unused prod `children` here, and you hardcoded placeholder value here instead of passing it in `children` prop